src/main/java/sim/coffee/OrderList.java

```diff
@@ -77,12 +79,22 @@ public Order get(int index) {
 		return orders.get(index);
 	}
 
-
+	// getTotalIncome for the day only. Not the entire history and entire list.
 	public BigDecimal getTotalIncome() {
+
+		// create variable to be returned after looping through orders
 		BigDecimal sum = new BigDecimal("0");
 
+		// Today's Date without time by using date from the last order placed
+		LocalDate dateToday = orders.get(orders.size()-1).getDate();
```

Comment:
Just to make this more explicit there's a static method `LocalDateTime.now()` you could use

Reply:
Using the LocalDateTime.now will cause the tests to fail - testBasket.gettotalIncome() will treturn 0 since the condition was set to only take orders from today, which isn't the case for setupOrder().

Reply:
That sounds like an issue with the test then, because the logic here makes more sense using that method (e.g. if the list is empty this could would fail, or if nothing is ordered you'd get the report for yesterday)